colorpairs.h

```diff
@@ -0,0 +1,41 @@
+#include <stdio.h>

```

Comment:
Good try in segregating code :+1: 
There is still scope for improvement like keeping declaration in .h files and definition in .c files